cmd/fmt_test.go

```diff
@@ -153,3 +153,62 @@ p {
 		}
 	})
 }
+
+func TestFmtFailFileChanges(t *testing.T) {
```

Comment:
❓ The test name confuses me -- this one is _not_ changing the format, is it? So we expect _no error_.

Reply:
Good catch I got the names reversed, will update.